server/server.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const uuid = require('uuid');
+const app = express();
+const port = 3000;
+const cors = require("cors");
+
+app.use(express.json());
+app.use(cors());
+
+const generateUniqueId = () => {
+  return uuid.v4();
+};
+
+const todoTasks = [
+  {
+    id: `${generateUniqueId()}`,
+    details: { task: "Wash the dishes", completed: false, deadline: "05/16/2024" }
```

Comment:
hi Alex 
probably just a minor suggestion about the date formats. 
"05/16/2024" looks like localized date, and apparently the date format is "MM/DD/YYYY" 
best option would be using ISO DATE format: "YYYY-MM-DD" 
is it possible to use ISO format for storing dates? 
and if needed, at the front-end side dates could be transformed to localised dates.
in ideal world we would like to store dates in ISO format and UTC time zone.